.gitignore

```diff
@@ -98,3 +98,6 @@ bagitobjecttransfer/static/
 .dockerenv
 *.sqlite3
 env/
+package.json
```

Comment:
Is there a reason why package.json is being ignored? It's committed in this merge request already

Reply:
No, I must've added that by mistake.